src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.cs

```diff
@@ -47,6 +47,8 @@ public async Task ItemsSourceDoesNotLeak()
 			WeakReference weakReference = null;
 			var collectionView = new CollectionView
 			{
+				Header = new Label { Text = "Header" },
+				Footer = new Label { Text = "Footer" },
```

Comment:
Can you put `#if !ANDROID` around these two lines with a comment what's going on for now?

Reply:
The test passes

Reply:
Oh it's a random other test, I would ignore it here, then.

Reply:
I don't see how changing Windows-only code would break Android tests?

Reply:
I made the test more robust in by adding the header and footer. In doing so it turns out that Android is leaking

Reply:
Created this issue to follow up on Android: https://github.com/dotnet/maui/issues/16961